client/components/sidebar/translations/Messages_fr_FR.xml

```diff
@@ -55,6 +56,7 @@
    <translation id="cloud_sidebar_actions_menu_dbaas_ts" qtlid="383525">Metrics</translation>
    <translation id="cloud_sidebar_actions_menu_paas_cda" qtlid="355044">Cloud Disk Array</translation>
    <translation id="cloud_sidebar_actions_menu_paas_veeam" qtlid="404113">Veeam CC</translation>
+   <translation id="cloud_sidebar_actions_menu_paas_veeam_enterprise" qtlid="404113">Veeam Enterprise</translation>
```

Comment:
You have to remove this `qtlid` because it doesn't refer to `Veeam Enterprise` but `Veeam CC`. Correct me if I'm wrong.

Reply:
done